slices/php8.3-common.yaml

```
@@ -0,0 +1,77 @@
+package: php8.3-common
+
+essential:
+  - php8.3-common_copyright
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      - libffi8_libs
+      - libssl3t64_libs
+      - php8.3-common_config
+      - ucf_bins
+    contents:
+      /usr/lib/php/20230831/calendar.so:
```

```suggestion
      /usr/lib/php/*/calendar.so:
      /usr/lib/php/*/ctype.so:
      /usr/lib/php/*/exif.so:
      /usr/lib/php/*/ffi.so:
      /usr/lib/php/*/fileinfo.so:
      /usr/lib/php/*/ftp.so:
      /usr/lib/php/*/gettext.so:
      /usr/lib/php/*/iconv.so:
      /usr/lib/php/*/pdo.so:
      /usr/lib/php/*/phar.so:
      /usr/lib/php/*/posix.so:
      /usr/lib/php/*/shmop.so:
      /usr/lib/php/*/sockets.so:
      /usr/lib/php/*/sysvmsg.so:
      /usr/lib/php/*/sysvsem.so:
      /usr/lib/php/*/sysvshm.so:
      /usr/lib/php/*/tokenizer.so:
```
Same comment as `slices/php8.3-opcache.yaml`

---

This would cause a conflict with readline and opcache. I already tried and fails installation.

Also suggested at https://github.com/canonical/chisel-releases/pull/593#discussion_r2216630709